transport/src/main/java/com/distkv/drpc/api/PbRequestDelegate.java

```diff
@@ -0,0 +1,93 @@
+package com.distkv.drpc.api;
+
+import com.distkv.drpc.codec.generated.DrpcProtocol;
+import com.google.protobuf.Any;
+import com.google.protobuf.Any.Builder;
+import com.google.protobuf.Message;
+
+public class PbRequestDelegate implements Request {
+
+  private DrpcProtocol.Request.Builder builder;
+  private DrpcProtocol.Request delegatedRequest;
+
+  public PbRequestDelegate() {
+    builder = DrpcProtocol.Request.newBuilder();
+  }
+
+  public PbRequestDelegate(DrpcProtocol.Request delegatedRequest) {
+    this.delegatedRequest = delegatedRequest;
+  }
+
+  @Override
+  public long getRequestId() {
+    if (delegatedRequest == null) {
```

Comment:
should this be null?

Reply:
Might be, if 'build' is not not be invoked, 'delegatedRequest' is null. 
As using delegatedRequest without invoke 'build' before is a bad move, I think that maybe we could throw a 'IllegalStateException' to alarm that the 'build method' has not been invoked. I finally choose to test if delegatedRequest is null to avoid 'npe', but it indeed sallow lots of Exceptions.
So I'm not so sure about this place.

Reply:
While pb is using builder pattern, I can's just invoke a set-method to set a value without invoke a 'build-method', so the delegator-object must support build method.

Reply:
Thanks, I see.

Reply:
Thanks, I see.